tools/htdocs/components/20_VRForm.js

```diff
@@ -0,0 +1,447 @@
+/*
+ * Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+ * Copyright [2016-2018] EMBL-European Bioinformatics Institute
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ensembl.Panel.VRForm = Ensembl.Panel.ToolsForm.extend({
+
+  init: function() {
+    this.base();
+
+    this.resetSpecies(this.defaultSpecies);
+
+    this.consequencesData = this.params['consequences_data'];
+    delete this.params['consequences_data'];
+
+    this.exampleData = this.params['example_data'];
+    delete this.params['example_data'];
+
+    // this.autocompleteData = this.params['plugin_auto_values'];
+    // delete this.params['plugin_auto_values'];
+
+    var panel = this;
+
+    // Change the input value on click of the examples link
+    this.elLk.form.find('a._example_input').on('click', function(e) {
+      e.preventDefault();
+
+      var species = panel.elLk.form.find('input[name=species]:checked').val() || panel.elLk.form.find('input[name=species]').val();
+      var text    = panel.exampleData[species][this.rel];
+      if (typeof text === 'undefined' || !text.length) {
+        text = '';
+      }
+      text = text.replace(/\\n/g, "\n");
+
+      panel.elLk.dataField.val(text).trigger('focus');
+    });
+
+    // Preview button
+    this.elLk.previewButton = panel.elLk.form.find('[name=preview]').hide().on('click', function(e) {
+      e.preventDefault();
+      panel.enablePreviewButton(false);
+      panel.preview($(this).data('currentVal'), { left : e.pageX, top : e.pageY });
+    });
```

Comment:
This can be deleted, specific to VEP which is the run instant ... button

Reply:
This code seems to be used somewhere. If removed, the option to Edit & resubmit job doesn't work.

Reply:
Thanks for doing this @dglemos , it shouldn't be used somewhere else, if it is then probably something else is broken. I think its best we do like video call so that i can see where the error is coming from.